test/presentation/feature/news/top_anime_page_test.dart

```diff
@@ -25,7 +25,7 @@ Future<void> _loadTopAnimePage(WidgetTester tester) async {
             .overrideWithValue(_mockMyAnimeListRepository)
       ],
       child: const MaterialApp(
-        home: TopAnimePage(),
```

Comment:
Please restore the TopAnimePage (why was it deleted?), and add the RecordLecture page as an extra tab.